Godeps/Godeps.json

```diff
@@ -104,6 +104,10 @@
 			"Rev": "ee2372b58cee877abe07cde670d04d3b3bac5ee6"
 		},
 		{
+			"ImportPath": "github.com/rakyll/pb",
```

Comment:
oh... i think it is the bench tool. but are we building the bench tool in the build script?